samples/LibVLCSharp.NetCore.Sample/LibVLCSharp.NetCore.Sample.csproj

```diff
@@ -1,10 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework Condition="$([MSBuild]::IsOsPlatform('Windows'))">net6.0</TargetFramework>
+    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Windows'))">net6.0;net6.0-windows;net6.0-macos</TargetFrameworks>
     <TargetFramework Condition="!$([MSBuild]::IsOsPlatform('Windows'))">netcoreapp3.1</TargetFramework>
-    <Platforms>AnyCPU;x64;x86</Platforms>
```

Comment:
not sure we should remove these, a `net6.0-windows` will by default be AnyCPU, but what if you want to test the `x64` and `x86` loading ?

If the goal is to test the initialization of the core in the cross-product of all platforms combination, this should be kept.

Reply:
Sure